### PR TITLE
Redesigned contributions guidance

### PR DESCRIPTION
**Pull request by tomyems**
Renamed the get involved section, to contribute.
Removed lots of unnecessary information.
Focussed on encouraging people to add to the discussions.

### CODE REVIEW

**Comment by tomyems** [comment]
Change create an account and sign in link to include the return_to, to direct user back to the discussions. For example: sign in - https://github.com/login?return_to=https%3A%2F%2Fgithub.com%2FUKHomeOffice%2Fdesign-system%2Fdiscussions and sign up - https://github.com/signup?ref_cta=Sign+up&ref_loc=header+logged+out&ref_page=%2F%3Cuser-name%3E%2F%3Crepo-name%3E%2Fdiscussions%2Findex&source=header-repo&source_repo=UKHomeOffice%2Fdesign-system

**Comment by daniel-ac-martin** [comment]
I think it might be worth including some more information. Such as:
- The existence of the Slack channel
- The existence of the working group? Along with an e-mail address? (Simplify?: https://design.homeoffice.gov.uk/get-involved/working )
- Might be worth having a new version of https://design.homeoffice.gov.uk/get-involved/githubguide but for discussions? (Probably not urgent.)
- https://design.homeoffice.gov.uk/get-involved/proving might be useful but probably doesn't belong in this section.

**Comment by daniel-ac-martin** [comment]
And I should add some more advanced, technical stuff for developer contributions. (But that's not urgent either.)

**Comment by daniel-ac-martin** [comment]
I could probably adapt https://not-gov.uk/contributing for the dev guidance.

File: apps/docs/src/common/pages/contribute.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    <h3>1. Check if there's already a discussion</h3>
+    <p><a href="https://github.com/UKHomeOffice/design-system/discussions">Check if there are any open discussions</a> about your suggestion on the Design System Github.</p>
+    <h3>2. Create a new discussion</h3>
+    <p>If there is not a discussion, <a href="https://github.com/UKHomeOffice/design-system/discussions/new">start a new discussion</a> on github. Select ‘ideas’ from the ‘select category’ list when starting to create the discussion.</p>
+    <p>For help, use our <a href="/contribute/discuss-on-github">github guide</a> or ask the <a href="/contribute/design-system-working-group">working group</a>.</p>
+    <p>When discussing a proposal for a new component, try to explain why it should be included in our design system. If you can include screenshots and research findings.</p>
+    <h3>3. Share the discussion</h3>
+    <p>Let the UCD and Accessibility community know that you have posted a new discussion and ask their comments.</p>
+    <p>Share your discussion through slack, show and tells and community meetups.</p>
+    <p>Consider give your discussion topic a deadline of a few weeks.</p>
```
Comment (Sarraounia) [inline apps/docs/src/common/pages/contribute.tsx:60]: Replace 'give' with 'giving' so sentence reads more fluently.

File: apps/docs/src/common/pages/contribute/design-system-working-group.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        <li>decide when a discussion is ready to become a pattern or component in the design system</li>
+        <li>identify any discussion that are covered by the GOV.UK design system and raise it with them</li>
+      </ul>
+      <p>The working group will also:</p>
+      <ul>
+        <li>move discussions to <a href="https://github.com/UKHomeOffice/design-system/issues">issues to be worked on by the working group</a></li>
+        <li>publish and update the Home Office design system</li>
+        <li>share updates about the Home Office design system and encourage the UCD community to get involved</li>
+      </ul>
+      <p>You can contact the working group:</p>
```
Comment (Sarraounia) [inline apps/docs/src/common/pages/contribute/design-system-working-group.tsx:41]: Should we be clear about what we'd want people to contact about? For example:

'If you have a question or idea you can contact the working group:'

File: apps/docs/src/common/pages/contribute/discuss-on-github.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        <li>ask questions about a style, component or pattern</li>
+        <li>answer questions from others</li>
+        <li>share examples or demos of a component or pattern</li>
+        <li>share research relating to a style, compontent or pattern</li>
+      </ul>
+      <h2>Create a new discussion</h2>
+      <p>Your new discussion can be as simple as a thought you've had, or more well-formed research findings.</p>
+      <ol>
+        <li>Create a <a href="https://github.com/UKHomeOffice/design-system/discussions/new">new discussion</a>.</li>
+        <li>Select the Ideas category.</li>
```
Comment (Sarraounia) [inline apps/docs/src/common/pages/contribute/discuss-on-github.tsx:42]: This should follow the same format as previous quotes from GitHub on the 'Contribute to the design system' page:

Select the 'ideas' category.

File: apps/docs/src/common/pages/contribute.tsx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    </ul>
+    <h2>Make a suggestion</h2>
+    <h3>1. Check if there's already a discussion</h3>
+    <p><a href="https://github.com/UKHomeOffice/design-system/discussions">Check if there are any open discussions</a> about your suggestion on the Design System Github.</p>
+    <h3>2. Create a new discussion</h3>
+    <p>If there is not a discussion, <a href="https://github.com/UKHomeOffice/design-system/discussions/new">start a new discussion</a> on github. Select ‘ideas’ from the ‘select category’ list when starting to create the discussion.</p>
+    <p>For help, use our <a href="/contribute/discuss-on-github">github guide</a> or ask the <a href="/contribute/design-system-working-group">working group</a>.</p>
+    <p>When discussing a proposal for a new component, try to explain why it should be included in our design system. If you can include screenshots and research findings.</p>
+    <h3>3. Share the discussion</h3>
+    <p>Let the UCD and Accessibility community know that you have posted a new discussion and ask their comments.</p>
```
[the remark this answers is not in the document]
Comment (tomyems) [reply apps/docs/src/common/pages/contribute.tsx]: Updated.

File: apps/docs/src/common/pages/contribute.tsx [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+      <li>share examples or demos of a component or pattern</li>
+      <li>share research relating to a style, compontent or pattern</li>
+    </ul>
+    <h2>Make a suggestion</h2>
+    <h3>1. Check if there's already a discussion</h3>
+    <p><a href="https://github.com/UKHomeOffice/design-system/discussions">Check if there are any open discussions</a> about your suggestion on the Design System Github.</p>
+    <h3>2. Create a new discussion</h3>
+    <p>If there is not a discussion, <a href="https://github.com/UKHomeOffice/design-system/discussions/new">start a new discussion</a> on github. Select ‘ideas’ from the ‘select category’ list when starting to create the discussion.</p>
+    <p>For help, use our <a href="/contribute/discuss-on-github">github guide</a> or ask the <a href="/contribute/design-system-working-group">working group</a>.</p>
+    <p>When discussing a proposal for a new component, try to explain why it should be included in our design system. If you can include screenshots and research findings.</p>
```
[the remark this answers is not in the document]
Comment (tomyems) [reply apps/docs/src/common/pages/contribute.tsx]: Updated.

File: apps/docs/src/common/pages/contribute.tsx [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+    <h3>3. Share the discussion</h3>
+    <p>Let the UCD and Accessibility community know that you have posted a new discussion and ask their comments.</p>
+    <p>Share your discussion through slack, show and tells and community meetups.</p>
```
[the remark this answers is not in the document]
Comment (tomyems) [reply apps/docs/src/common/pages/contribute.tsx]: Updated

File: apps/docs/src/common/pages/contribute/design-system-working-group.tsx [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+      <h1>
+        {title}
+      </h1>
+      <p>The Home Office design system working group is a cross-functional team. It’s made up of interaction and content design, user research, accessibility, and front-end development. </p>
+      <p>They meet every 2 weeks where they will:</p>
+      <ul>
+        <li>review new discussions and whether something else better meets the need</li>
+        <li>check how discussions are progressing and if it needs support from the group</li>
+        <li>decide when a discussion is ready to become a pattern or component in the design system</li>
+        <li>identify any discussion that are covered by the GOV.UK design system and raise it with them</li>
```
[the remark this answers is not in the document]
Comment (tomyems) [reply apps/docs/src/common/pages/contribute/design-system-working-group.tsx]: Updated.

File: apps/docs/src/common/pages/contribute/design-system-working-group.tsx (hunk 2 above)
Comment (tomyems) [reply apps/docs/src/common/pages/contribute/design-system-working-group.tsx]: Updated.

File: apps/docs/src/common/pages/contribute/discuss-on-github.tsx (hunk 3 above)
Comment (tomyems) [reply apps/docs/src/common/pages/contribute/discuss-on-github.tsx]: Updated.

File: apps/docs/src/common/pages/contribute/discuss-on-github.tsx [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+    </div>
+    <div className="govuk-grid-column-three-quarters">
+      <h1>
+        {title}
+      </h1>
+      <p>Our <a href="https://github.com/UKHomeOffice/design-system/discussions">github repository discussions</a> is where our designs are discussed and problems explored.</p>
+      <h2>Create an account</h2>
+      <p><a href="https://github.com/signup?return_to=https%3A%2F%2Fgithub.com%2FUKHomeOffice%2Fdesign-system%2Fdiscussions&source=login">Create a github account</a> with any email address, including your Home Office email addresses.</p>
+      <h2>Join a discussion</h2>
+      <p>If there's an existing discussion contribute. For example, you can:</p>
```
[the remark this answers is not in the document]
Comment (tomyems) [reply apps/docs/src/common/pages/contribute/discuss-on-github.tsx]: Updated

File: apps/docs/src/common/pages/contribute.tsx [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+import { PageProps } from '@not-govuk/app-composer';
+import { A, NavigationMenu } from '@not-govuk/components';
+
+export const menu = (
+  <Fragment>
+    <NavigationMenu items={[
+          {
+            href: '/contribute',
+            text: 'Contribute to the design system'
+          },
```
Comment (daniel-ac-martin) [inline apps/docs/src/common/pages/contribute.tsx:12]: Elsewhere we don't have a link on the side to the main page. We just have the link in the main, horizontal navigation. So I think this should go, just so we are consistent.

File: apps/docs/src/common/pages/contribute/design-system-working-group.tsx [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+      <meta name="og:title" content={title} />
+      <meta name="og:description" content={description} />
+      <meta name="og:article:section" content={section} />
+    </Helmet>
+    <div className="govuk-grid-column-one-quarter">
+      {menu}
+    </div>
+    <div className="govuk-grid-column-three-quarters">
+      <h1>
+        {title}
```
Comment (daniel-ac-martin) [inline apps/docs/src/common/pages/contribute/design-system-working-group.tsx:25]:
```suggestion
        <span className="caption">{section}</span>
        {title}
```
On the other sub-pages we include a caption for the overall section. (So 'Contribute' in this case.)

File: apps/docs/src/common/pages/contribute/discuss-on-github.tsx [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+      <meta name="og:title" content={title} />
+      <meta name="og:description" content={description} />
+      <meta name="og:article:section" content={section} />
+    </Helmet>
+    <div className="govuk-grid-column-one-quarter">
+      {menu}
+    </div>
+    <div className="govuk-grid-column-three-quarters">
+      <h1>
+        {title}
```
Comment (daniel-ac-martin) [inline apps/docs/src/common/pages/contribute/discuss-on-github.tsx:25]:
```suggestion
        <span className="caption">{section}</span>
        {title}
```

File: apps/docs/src/common/pages/contribute.tsx (hunk 9 above)
Comment (tomyems) [reply apps/docs/src/common/pages/contribute.tsx:12]: Ah yes. I'd forgotten about this. I have been confused by this, as not known how to get back to the main page, before realising to select the main navigation again. Also having more than one link on a page to the same page can be confusing. So I've removed for now, if we get repeated feedback about this confusion then we should look at this again - but if we do change it, we should be consistent across the main pages.

File: apps/docs/src/common/pages/contribute/discuss-on-github.tsx (hunk 11 above)
Comment (tomyems) [reply apps/docs/src/common/pages/contribute/discuss-on-github.tsx:25]: Done

**Review by daniel-ac-martin — APPROVED** [review]

**Pull request merged**